<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>707</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mini Minecraft</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="scenesWidget">
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>2000</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="game">
       <widget class="MyGL" name="mygl">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1011</width>
          <height>661</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>2000</width>
          <height>2000</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="title">
       <widget class="QPushButton" name="singleplayerButton">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>190</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Singleplayer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="multiplayerButton">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>250</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Multiplayer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>60</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>MINECRAFT</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="usernameText">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>140</y>
          <width>120</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Technoblade</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>120</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="multiplayer">
       <widget class="QLineEdit" name="serverIPText">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>150</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Server IP</string>
        </property>
       </widget>
       <widget class="QPushButton" name="joinServerButton">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>149</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>-&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>60</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Multiplayer</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>120</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Join a server</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>210</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Host a server</string>
        </property>
       </widget>
       <widget class="QPushButton" name="hostServerButton">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>200</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>-&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="portText">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>280</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>3080</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>250</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Port #</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1170</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCamera_Controls"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCamera_Controls">
   <property name="text">
    <string>Camera Controls</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGL</class>
   <extends>QOpenGLWidget</extends>
   <header>mygl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
